graphstorm-processing/tests/test_dist_numerical_transformation.py

```diff
@@ -453,3 +458,91 @@ def test_rank_gauss_reshuffling(spark: SparkSession, check_df_schema, epsilon):
         assert_almost_equal(
             [row["rand"]], expected_vals[i, :], decimal=4, err_msg=f"Row {i} is not equal"
         )
+
+
+def test_json_representation(input_df: DataFrame, check_df_schema):
+    """Test that the generated representation is correct"""
+    dist_numerical_transformation = DistNumericalTransformation(
+        ["salary", "age"], imputer="mean", normalizer="min-max"
+    )
+    transformed_df = dist_numerical_transformation.apply(input_df)
+    json_rep = dist_numerical_transformation.get_json_representation()
+
+    assert "cols" in json_rep
+    assert "imputer_model" in json_rep
+    assert "normalizer_model" in json_rep
+    assert "out_dtype" in json_rep
+    assert json_rep["cols"] == ["salary", "age"]
+    assert json_rep["imputer_model"]["imputer_name"] == "mean"
+    assert json_rep["normalizer_model"]["norm_name"] == "min-max"
+
+    check_df_schema(transformed_df)
+
+
+def test_precomputed_transformation(input_df: DataFrame, check_df_schema):
```

Comment:
Is this test covered by test_precomputed_normalization?

Reply:
Not exactly the other two tests set either imputer or normalizer to `'none'`